test/source/tests/page-recipe/compose-page-recipe.ts

```diff
@@ -85,9 +85,6 @@ export class ComposePageRecipe extends PageRecipe {
   }
 
   public static fillRecipients = async (composePageOrFrame: Controllable, recipients: Recipients, windowType: 'new' | 'reply') => {
-    if (windowType === 'reply') { // new messages should already have cc/bcc buttons visible, because they should have recipients in focus
-      await composePageOrFrame.waitAndClick('@action-show-container-cc-bcc-buttons');
```

Comment:
Focus is already in `@input-to`, no need to click the placeholder `@action-show-container-cc-bcc-buttons` in test.